tools/prom-metrics-collector/metrics_collector.go

```diff
@@ -26,14 +26,9 @@ import (
 	dto "github.com/prometheus/client_model/go"
 )
 
-// excludedMetrics defines the metrics to ignore,
-// open issue:https://github.com/kubevirt/containerized-data-importer/issues/2773
-// Do not add metrics to this list!
-var excludedMetrics = map[string]struct{}{
-	"clone_progress":                                {},
-	"kubevirt_cdi_operator_up_total":                {},
-	"kubevirt_cdi_incomplete_storageprofiles_total": {},
-}
+// This should be used only for very rare cases where the naming conventions that are explained in the best practices
```

Comment:
Please add the link to the best practices.

Reply:
Right, BTW in kubevirt the link added.

Reply:
Done